hedera-node/src/test/java/com/hedera/services/ServicesStateTest.java

```diff
@@ -257,14 +272,25 @@ public void initsAsExpected() {
 	}
 
 	@Test
-	public void lookupForContext() {
+	public void initializesContext() {
+		InOrder inOrder = inOrder(ctx, txnHistories, historian, systemFilesManager);
+
 		given(ctx.nodeAccount()).willReturn(AccountID.getDefaultInstance());
+		// and:
 		CONTEXTS.store(ctx);
 
+		// when:
 		subject.init(platform, book);
-		InOrder inOrder = inOrder(ctx);
+
+		// then:
 		inOrder.verify(ctx).nodeAccount();
 		inOrder.verify(ctx).update(subject);
+		inOrder.verify(ctx).rebuildBackingStoresIfPresent();
+		inOrder.verify(historian).reviewExistingRecords();
+		inOrder.verify(systemFilesManager).loadApplicationProperties();
+		inOrder.verify(systemFilesManager).loadApiPermissions();
+		// TODO: uncomment below line.
+		// inOrder.verify(systemFilesManager).loadFeeSchedules();
```

Comment:
this line could be uncommented, right?

Reply:
@QianSwirlds you are right. I just uncommented it.